### DISCUSSION TITLE
Jmeter benchmark:Apisix VS Nginx, CPU occupancy rate huge?

### DISCUSSION THREAD

**Discussion by Git4Future** [apache/apisix#6731] [Show and tell]
We run another Nginx to listen 81 port,and it returns statics files response like 'helo.html'
this is my route config:
`{
  "uri": "/*",
  "name": "local-body",
  "methods": [
    "GET",
    "POST"
  ],
  "host": "local-body.test.cn",
  "upstream": {
    "nodes": [
      {
        "host": "10.0.1.94",
        "port": 81,
        "weight": 1
      }
    ],
    "timeout": {
      "connect": 6,
      "send": 6,
      "read": 6
    },
    "type": "roundrobin",
    "scheme": "http",
    "pass_host": "pass",
    "keepalive_pool": {
      "idle_timeout": 75,
      "requests": 10000,
      "size": 1000
    }
  },
  "status": 1
}`
**Environment:**
APISIX:  10.0.1.98  4C
NGINX:  10.0.1.19.  4C
JMETER:8C
NGINX UPSTREAM: 10.0.1.94 4C
 
**This is the result from apisix:**
![t4_100_100k](https://user-images.githubusercontent.com/24261816/160322652-82afebf7-3557-4225-85a9-744909f32319.png)
**This is the result from nginx:**
![T6](https://user-images.githubusercontent.com/24261816/160559148-900c565f-720e-4189-aa98-d88913ef9c48.png)
why CPU occupancy rate between Apisix and Nginx are so huge?

**Comment by tzssangglass** [comment]
You can make a CPU flamegraph to see where the CPU consumption is
